en/docs/deploy/migrate/what-has-changed.md

```diff
@@ -657,4 +657,36 @@ In WSO2 Identity Server 6.0.0, the Log4j version is upgraded to ```Log4j2```. Th
 ``` js
 [hazelcast]
 hazelcast.logging.type = log4j2
+```
+### Change In Data Type for Boolean Attributes From Schemas Endpoint
+
+`scim2/Schemas` endpoint now returns the attribute type of boolean attributes as `boolean`. Before this fix, the attribute type was returned as `string`.
```

Comment:
Let's stick to the same format that we have been using. (i.e., first, we mention the previous version, and then the update)
```suggestion
In previous versions of WSO2 Identity Server, the `scim2/Schemas` endpoint returned the attribute type of boolean attributes as `string`. 

In Identity Server 6.0.0, the attribute type returned for boolean attributes of the `scim2/Schemas` endpoint is `boolean`.
```